### PR TITLE
changed button text

### PR DESCRIPTION
**Pull request by jllamasi**
Hi there! I just did what issue #275 requested! Hope everything's okay!

### CODE REVIEW

File: i18n/en.i18n.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
 
 
   // buttons
+  "save_changes": "Save changes",
```
Comment (camilleriluke) [inline i18n/en.i18n.json]: Since `edit_a_hangout` is not used anymore it can be deleted

Comment (jllamasi) [inline i18n/en.i18n.json]: @camilleriluke it is still used as modal title

Comment (camilleriluke) [inline i18n/en.i18n.json]: You are right :) my bad.

**Comment by lpatmo** [comment]
Thanks @jllamasi!

**Pull request merged**